recipes/cryptoauthlib/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "cryptoauthlib" %}
+{% set version = "20210118" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cryptoauthlib-{{ version }}.tar.gz
+  sha256: 8a439ff9934eb0071dd23a089da17d30d075c7db81c8b36fa639ee3feea0e8a2
+
+build:
+  number: 0
+  skip: true  # [linux or py>=40]
```

As Python 4 doesn't exist yet so the compatibility isn't known and it would be applied as a migration in any case so you could opt in/out at that time:
```suggestion
  skip: true  # [linux]
```